src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -93,6 +94,7 @@ public void InitializeService(ServiceHost serviceHost)
             this.ServiceHost = serviceHost;
             this.ServiceHost.SetRequestHandler(MigrationAssessmentsRequest.Type, HandleMigrationAssessmentsRequest);
             this.ServiceHost.SetRequestHandler(ValidateWindowsAccountRequest.Type, HandleValidateWindowsAccountRequest);
+            this.ServiceHost.SetRequestHandler(ValidateNetworkFileShareRequest.Type, HandleValidateNetworkFileShareRequest);
```

Comment:
Not sure if I'm missing anything here but I'm a bit confused that why do we need both `ValidateWindowsAccountRequest` and `ValidateNetworkFileShareRequest` for network share validation. I assume as long as the `{UNC, username, password}` can access the network share, we should be good, is this correct?

Reply:
These validations are used in 2 separate places in the extension. 

Validate windows account is used in this screen. We are trying to get the windows account password for the Integrated sql server connection used in migration. This has nothing do with File share. 
![image](https://user-images.githubusercontent.com/6816294/117059060-3fe5c880-acd4-11eb-8fc1-c59873f80233.png)

Validate network file share is used in the screen where the user provides an SMB file share that contains the sql server backup files 
![image](https://user-images.githubusercontent.com/6816294/117059048-3a887e00-acd4-11eb-9d3a-7ed0a147a8f0.png)

Reply:
This makes sense now. It might be better to leave the changes related to windows auth to a separate PR but no biggie.